src/qibo_cloud_backends/__init__.py

```diff
@@ -4,12 +4,13 @@
 
 from qibo.config import raise_error
 
+from qibo_cloud_backends.braket_client import BraketClientBackend
 from qibo_cloud_backends.qibo_client import QiboClientBackend
 from qibo_cloud_backends.qiskit_client import QiskitClientBackend
 
 __version__ = im.version(__package__)
 
-QibocloudBackend = Union[QiboClientBackend, QiskitClientBackend]
+QibocloudBackend = Union[QiboClientBackend, QiskitClientBackend, BraketClientBackend]
```

Comment:
I just realized that we need to add the `BraketClientBackend` to the `MetaBackend.list_available()` function below. 
You'll need to add the `braket-client` element to the `CLIENTS` list below and probably a `None` placeholder to the `TOKENS` as `braket` does not need one. This also implies that the `list_available` function has to be slightly changed to handle the `braket-client`.